File: algorithms/T/Three Consecutive Odds/Three Consecutive Odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int count = 0,n = arr.length;
        for(int i=0;i<n;i++)
        {
            if((arr[i] & 1) > 0)
            {
                count++;
                if(count == 3) return true;
            }
            else
            {
                count = 0;
            }
        }
        return false;
    }
}

File: scripts/algorithms/T/Three Consecutive Odds/Three Consecutive Odds.py
# Runtime: 106 ms (Top 5.05%) | Memory: 14 MB (Top 60.17%)
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        c=0
        for i in arr:
            if i%2==0:
                c=0
            else:
                c+=1
                if c==3:
                    return True
        return False